docs/pages/installation.mdx

```diff
@@ -107,21 +107,216 @@ information on obtaining Teleport binaries compatible with Teleport Cloud.
 
 ## Docker
 
+### Images
+
+We provide a pre-built Docker image for every version of Teleport. This section
+describes the available Docker images.
+
+These images are hosted on [Amazon ECR
+Public](https://gallery.ecr.aws/gravitational).
+
+#### Image suffixes
+
+For each of the image names listed in this section, you can specify attributes
+of the image by appending a suffix to the repository name or tag.
+
+Images with the `*-distroless` suffix within the repository name include only
+the `teleport` binary and its runtime dependencies, with no shell or utility
+applications. An example is `public.ecr.aws/gravitational/teleport-distroless` for Teleport
+Community Edition.
+
+Images with the `*-distroless-debug` suffix within the repository name include a
+Busybox shell and tool suite in addition to Teleport, and are intended for
+troubleshooting deployments only. They are not intended for production use. An
+example is `public.ecr.aws/gravitational/teleport-distroless-debug`.
+
+You can specify the architecture of an image by appending a suffix to its tag.
+We support the following architecture suffixes: `amd64`, `arm`, and `arm64`. For
+example, if you want to pull the ARM64 image for
+`public.ecr.aws/gravitational/teleport`, you can use
+`public.ecr.aws/gravitational/teleport:(=teleport.version=)-arm64`. Architecture
+suffixes are not available for `*-distroless` or `*-distroless-debug` images.
+
+#### Nightly image builds
+
+Nightly images point to the latest version of Teleport. They are stable, and we
```

Comment:
I'm unfamiliar with these.

Why are these called nighty? Do we rebuild them every night? If so, do we really recommend that?